src/azure-cli/azure/cli/command_modules/acs/_validators.py

```diff
@@ -256,6 +257,39 @@ def validate_taints(namespace):
                 raise CLIError('Invalid node taint: %s' % taint)
 
 
+def validate_priority(namespace):
+    """Validates the node pool priority string."""
+    if namespace.priority is not None:
+        if namespace.priority == '':
+            return
+        if namespace.priority != "Spot" and \
+                namespace.priority != "Regular":
+            raise CLIError("--priority can only be Spot or Regular")
```

Comment:
`--priority` can also be an empty string, right ?

Reply:
when get to here. It's already not none "if namespace.priority is not None:"